test/modules/packages/distributedIters/DistributedIters.chpl

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright 2004-2017 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /*
+   This module contains iterators that can be used to distribute a `forall`
+   loop for a range or domain.
+
+   Part of a 2017 Cray summer intern project by Sean I. Geronimo Anderson
+   (sgeronimo@cray.com) as mentored by Ben Harshbarger (bharshbarg@cray.com).
```

Comment:
This information could also just be in the merge message and live on in the git history.